tracks/python/exercises/matrix/mentoring.md

```diff
@@ -55,3 +55,26 @@ update an entry.
 
 In any case, this isn't a topic to burden a beginner with, 
 but you may want to comment about it if you see it in the solution. 
+
+### Separation of Concerns
+
+You will see solutions where the student has split `matrix_string` in the constructor, then done the actual construction of the matrix in the getters.
+
+```
+class Matrix:
+    def __init__(self, matrix_string):
+        self.matrix = matrix_string.split('\n')
+
+    def row(self, index):
+        return list(map(int, self.matrix[index - 1].split(' ')))
+
+    def column(self, index):
+        column = []
```

Comment:
Just for visual simplicity I'd implement both **row** and **column** using a single list comprehension; the point we're trying to make here is not parsing the matrix string itself in either method when it could be done in the `__init__`, and the two different implementations are distractions from that point.